P1: CRUD usage examples moved to appropriate pages (#618)
mongodb/docs-java: source/crud/write-operations/delete.txt

    { "_id": 1, "color": "red", "qty": 5 }
    { "_id": 8, "color": "black", "qty": 8 }

Delete Example: Full File
-------------------------

The following code is a complete, standalone file that performs a delete one
operation and a delete many operation. 

.. include:: /includes/crud/example-intro.rst

> The query uses the ``eq()`` and ``lt()`` filters to query documents. For more

Reviewer: S: same suggestion about keeping the code example description together and moving the /includes/crud/example-intro.rst

Reviewer: I'd also move the link to the Filters API docs to a tip admonition after the code example

    { "_id": 1, "color": "red", "qty": 5 }
    { "_id": 8, "color": "black", "qty": 8 }

Delete Example: Full File
-------------------------

.. include:: /includes/crud/example-intro.rst

The following code is a complete, standalone file that performs a delete one

> operation and a delete many operation. 

Reviewer: S:
```suggestion
operation and a delete many operation:
```

.. io-code-block::

   .. input:: /includes/crud/Delete.java
      :language: java
      :dedent:

   .. output:: 
      :language: none
      :visible: false

>       DeleteOne document count: 1
>       DeleteMany document count: 4

Reviewer: S: update the output to reflect the code changes
```suggestion
      deleteOne() document count: 1
      deleteMany() document count: 4
```

> The query in this examples uses the ``eq()`` and ``lt()`` filters to query documents. For more

Reviewer: S:
```suggestion
The queries in the preceding example use the ``eq()`` and ``lt()`` filters to query documents. For more
```
